Nero/test/testNero.py

```diff
@@ -495,9 +432,10 @@
                 process.egmPhotonIDSequence *
                 process.photonIDValueMapProducer * ## ISO MAP FOR PHOTONS
                 process.electronIDValueMapProducer * ## ISO MAP FOR PHOTONS
-		process.HBB * ## HBB 74X
-                process.jetSequence*
-		#process.jecSequence *
+            		process.HBB * ## HBB 74X
+                process.puppiSequence * ## does puppi, puppi met, type1 corrections
+                process.jetSequence *
+            		#process.jecSequence *
```

Comment:
It's always a bad idea to change the indentation inside the code commits. If you want to do so (for the future) the best way will be to do it: 1) when we have very little PR (that now will conflict) 2) making a PR/commit that changes only the indentation. 3) Align vim and emacs to be able to read the indentation with comments (# or // ) lines like these:
https://github.com/MiT-HEP/NeroProducer/blob/master/Core/interface/BareFunctions.hpp#L39-L45